src/registry/selectboxes/edit.tsx

```diff
@@ -27,9 +27,64 @@ import {LABELS} from '@/components/builder/messages';
 import {SelectBoxes, TabList, TabPanel, Tabs} from '@/components/formio';
 import {useErrorChecker} from '@/utils/errors';
 
+import {NumberField} from '../../components/formio';
 import {EditFormDefinition} from '../types';
 import {checkIsManualOptions} from './helpers';
 
+const MinSelectedCheckboxes: React.FC = () => {
```

Comment:
can you move this *under* the `EditForm` component implementation so that it's consistent with the rest of the codebase?